openshift-ci/Dockerfile.registry.build

```diff
@@ -8,8 +8,9 @@ COPY manifests manifests
 COPY deploy/crds/*.yaml manifests/devconsole/${version}/
 
 USER root
-RUN sed -e "s,REPLACE_IMAGE,registry.svc.ci.openshift.org/$(OPENSHIFT_BUILD_NAMESPACE)/stable:devconsole-operator," -i manifests/devconsole/${version}/devconsole-operator.v${version}.clusterserviceversion.yaml
+RUN ls /bin && sed -e "s,REPLACE_IMAGE,registry.svc.ci.openshift.org/${OPENSHIFT_BUILD_NAMESPACE}/stable:devconsole-operator," -i manifests/devconsole/${version}/devconsole-operator.v${version}.clusterserviceversion.yaml
+RUN /bin/initializer
```

Comment:
Should we add `USER 1001` before this?

Reply:
That's what I had, trying to see if that's what caused the issue.. hence moved it up..

Reply:
ok

Reply:
Now, Do you want to move this down again?